bucket/shadowsocksr-csharp.json

```diff
@@ -3,7 +3,7 @@
     "description": "An Internet censorship circumventing tunnel tool, forked by Breakwa11 from shadowsocks, maintained now by Akariiin",
     "version": "4.9.2",
     "license": "GPL-3.0-only",
-    "url": "https://github.com/shadowsocksrr/shadowsocksr-csharp/releases/download/4.9.2/ShadowsocksR-win-4.9.2.zip",
```

Comment:
Please add comment to manifest, that the zip rewrite to 7z is needed, so next maintainers will know about this. 
https://github.com/Ash258/scoop-Ash258/blob/master/bucket/ArchiSteamFarm.json#L2